watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py

```diff
@@ -200,11 +196,11 @@ def add_costing(m):
         / (
             m.fs.costing.utilization_factor
             * pyunits.convert(
-                m.fs.product_struvite.properties[0].flow_mass_comp["struvite"],
+                m.fs.product_salt.properties[0].flow_mass_comp["phosphorus"],
                 to_units=pyunits.kg / m.fs.costing.base_period,
             )
         ),
-        doc="Levelized cost of struvite",
+        doc="Levelized cost of phosphorus recovery",
```

Comment:
The `LCOS` variable no longer refers to levelized cost of struvite, so I recommend changing the name of the variable to something more appropriate. Perhaps `LCOP`.

Reply:
Good suggestion, modified!